packages/formspree-react/src/useForm.ts

```diff
@@ -122,40 +122,12 @@ const useForm = (
     }
 
     const createPaymentMethod = async () => {
-      const address = {
-        ...(formData.address_line1 && {
-          line1: formData.address_line1,
-        }),
-        ...(formData.address_line2 && {
-          line2: formData.address_line2,
-        }),
-        ...(formData.address_city && {
-          city: formData.address_city,
-        }),
-        ...(formData.address_country && {
-          country: formData.address_country,
-        }),
-        ...(formData.address_state && {
-          state: formData.address_state,
-        }),
-        ...(formData.address_postal_code && {
-          postal_code: formData.address_postal_code,
-        }),
-      };
-
```

Comment:
`formData` can be an instance of `FormData`. In such case, accessing these fields with dot notation returns `undefined` resulting to an empty object. We fix it in this PR.